modules/features/pages/catalog.adoc

```diff
@@ -1,3 +1,59 @@
+
+RHDH comes with a set of pre-built plugins that interact with the components in the catalog. For example, CI/CD pipelines, container images, issue tracking, Kubernetes cluster information and the deployments can all be viewed in RHDH without requiring developers to access these different tools using different applications, each with their own authentication and workflows. 
+
+NOTE: These plugins are not meant to replace the tooling, but to **complement** it, by reducing the __cognitive overload__ and preventing __context switching__ between different tools while letting the developer focus on building, deploying, releasing and maintaining software. The plugins can be configured to display limited, but **useful** information at a first glance. Links can be embedded in the plugin to navigate to the external tool or application in case the developer needs to investigate deeper. 
+
+image::rhdh-sw-catalog.png[title=Software Catalog in RHDH,width=800px]
+
+== Populating the Software Catalog
+
+The configuration of the catalog is done using YAML files, typically stored in source control systems like Git. These YAML files live alongside, and in the same source code repository as the source code of the software components. There are three ways to add components to the Software Catalog:
+
+=== Manual Component Registration
+
+To register an already existing component that is created outside RHDH, click on **Create** in the left sidebar navigation of RHDH, and then click **REGISTER NEW COMPONENT**.
+
+In addition to manually registering components, you can also register components through static configuration, by passing RHDH a YAML configuration file with the location and details about the component.
+
+image::rhdh-new-component.png[title=Manual Registration of Components,width=800px]
+
+=== Create Components from Golden Path Templates (GPT)
+
+A more flexible approach is to create components using RHDH **Golden Path Templates** (aka **Software Templates** in Backstage parlance), which automatically adds the components to the catalog. Using this approach, you can register multiple related components of an application at the same time with the Software Catalog.
+
+image::rhdh-new-gpt.png[title=Create Components using Golden Path Templates,width=800px]
+
+NOTE: Golden Path Templates (GPT) are covered in more detail in the next section.
+
+=== Third-party Integration
+
+RHDH can import software components from third-party external systems using its Typescript API. This would require you to extend the Typescript API of RHDH and is outside the scope of this course.
+
+The first two methods are the most common way to populate the catalog, with the third option being reserved for special scenarios where you already have an existing system that manages software components, and you want to integrate it with RHDH.
+
```

Comment:
There is another option to "dynamically discover" components.  The same dynamic discovery mentioned later in the GPTs can be used to discover components as well (or any Backstage entity).  It's like the "manual registration" in that you are registering an already existing component but Backstage does it for you instead of manually using the UI.

Reply:
Noted. Will add a short para to this section